src/store/product/index.js

```diff
@@ -0,0 +1,12 @@
+import { LOAD_PRODUCT } from './actions'
+
+const reducer = (state = [], action) => {
```

Comment:
product is object, so default state for it should be null

Reply:
Fixed!